.docker/parameters.yml

```
@@ -31,6 +31,9 @@ parameters:
     subject_rewrites: []
     example_rds_articles:
         '16846': https://repro.elifesciences.org/example.html
+    rds_articles:
+        '26231':
+            display: https://repro.elifesciences.org/example.html
```

```suggestion
            display: https://hub.stenci.la/elife/30274/main/
```
This is what we use in real configuration, and it was already an external URL anyway.
Currently get blocked for me with `Load denied by X-Frame-Options: “SAMEORIGIN” from “https://hub.stenci.la/elife/30274/main/”, site does not permit cross-origin framing from “http://localhost:8080/articles/26231/rds”.`